.bazelrc

```diff
@@ -25,6 +25,7 @@ build:android --cxxopt=-std=c++14
 build:android --host_cxxopt=-std=c++14
 build:ios --cxxopt=-std=c++14
 build:ios --host_cxxopt=-std=c++14
+build:ios --cxxopt=-xobjective-c++
```

Comment:
What is the reason to add this?

Reply:
Recent changes (between TF 2.6.0 and 2.8.0) in TensorFlow Lite GPU delegate code includes `<Metal/Metal.h>` which in turns contains Objective-C code. So we need this flag to compile `.cc` code which includes GPU metal delegate header.